mobile/src/components/pictureCardComponent.tsx

```diff
@@ -0,0 +1,29 @@
+import { StatusBar } from 'expo-status-bar';
+import React from 'react';
+import { StyleSheet, Image, View } from 'react-native';
+
+export default function Card() {
```

Comment:
I know this already got merged. but you should make the filename of the component the same as the component name if you want to follow convention.